mcc/Navigation.c

```diff
@@ -34,12 +34,17 @@ struct pos_info pos;
 LONG FlowSpace(struct InstData *data, UWORD flow, STRPTR text)
 {
   LONG flowspace = 0;
+  LONG textlength = TextLengthNew(&data->tmprp, text, LineCharsWidth(data, text)-1, data->TabSizePixels);
```

Comment:
just put `LONG textlength = 0;` variable definition here. and put assignment to TextLengthNew() after `ENTER()` debug statement or otherwise if TextLengthNew() crashes we won't be able to identify the crash location easily.